tensorflow/lite/micro/kernels/fully_connected_test.cc

```diff
@@ -435,6 +435,36 @@ TF_LITE_MICRO_TEST(SimpleTestQuantizedInt8) {
       kTfLiteOk);
 }
 
+#if defined(CMSIS_NN)
```

Comment:
It would be really great if we can also update the reference to support 16x8 and run test for both reference and CMSIS_NN.  Is that possible here?

Reply:
Done